spec/models/user_spec.rb

```diff
@@ -1,5 +1,13 @@
 require 'rails_helper'
 
 RSpec.describe User, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  it { should have_many :reviews }
+  it { should have_many :beers }
+
+  it { should have_valid(:email).when("dabomb@gmail.com", "hi@yahoo.com") }
+  it { should_not have_valid(:email).when(
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.